### PR TITLE
DE-294 Add merge tags for use as custom fields

### PR DESCRIPTION
**Pull request by paangaflo**
feat: [DE-294](https://makingsense.atlassian.net/browse/DE-294).

Basic configuration to add custom fields from local environment as to `merge tags` options unlayer.

Base documentation [here.](https://docs.unlayer.com/docs/merge-tags)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -30,10 +35,18 @@ const Bar = styled.div`
...
 }
 
+interface MergeTagsConfig {
+  readonly sort?: boolean | undefined;
+}
+
+interface UnlayerOptionsExtended extends UnlayerOptions {
+  readonly mergeTagsConfig?: MergeTagsConfig | undefined;
+}
+
```
Comment (andresmoschini) [inline src/App.tsx:49]: What do you think about simply use casting in place of extending the interfaces?

Comment (paangaflo) [reply src/App.tsx]: Corrected.

File: src/App.tsx [hunk 2]
```diff
@@ -42,10 +43,35 @@ const App: React.FC = () => {
...
   const userSignature: string = process.env.REACT_APP_USER_SIGNATURE as string;
-  const userExtend: UserSecurity = {
+  const userExtend = {
     id: userId,
     signature: userSignature,
-  };
+  } as User;
+  const UnlayerOptionsExtended = {
+    mergeTagsConfig: {
+      sort: true,
```
Comment (andresmoschini) [inline src/App.tsx:52]: What do you think about force the editor to make honor to the original order?

```suggestion
      sort: false,
```

Comment (paangaflo) [reply src/App.tsx]: Corrected.

Comment (andresmoschini) [reply src/App.tsx]: 🚀

**Pull request merged**